src/ray/raylet/node_manager.cc

```diff
@@ -2185,7 +2184,8 @@ void NodeManager::ScheduleTasks(
           << " for placement, however there are no nodes in the cluster that can "
           << "provide the requested resources. To resolve this issue, consider "
```

Comment:
Let's rewrite this entire message starting:

"... however the cluster currently cannot provide the requested resources. The required resources may be added as autoscaling takes place or placement groups are scheduled. Otherwise, consider reducing the resource requirements of the task."